app/src/main/java/de/westnordost/streetcomplete/data/osmnotes/CreateNoteDao.java

```diff
@@ -46,7 +46,10 @@ public boolean add(CreateNote note)
 			values.put(CreateNoteTable.Columns.IMAGE_PATHS, serializer.toBytes(note.imagePaths));
 		}
 		values.put(CreateNoteTable.Columns.TEXT, note.text);
-		values.put(CreateNoteTable.Columns.QUEST_TITLE, note.questTitle);
+		if (note.questTitle != null)
+		{
+			values.put(CreateNoteTable.Columns.QUEST_TITLE, note.questTitle);
```

Comment:
Does the table allow NULL there?

Reply:
Also, could questTitle being null cause a NPE someplace else?

Reply:
Ok nevermind this comment.

Reply:
Whoops, I meant to write that someplace else, *do* mind that comment!

Reply:
(BTW: You can delete comments…)

Reply:
> Does the table allow NULL there?

I think so... In the StreetCompleteOpenHelper it does not say ```NOT NULL```

> Also, could questTitle being null cause a NPE someplace else?

I don't think so... As far as I have seen, there are null-checks everywhere.